README.md

```diff
@@ -12,4 +12,4 @@ using hooks. Keep CSS classes for tests.
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Use the [React TypeScript cheat sheet](https://mate-academy.github.io/fe-program/js/extra/react-typescript).
 - Open one more terminal and run tests with `npm test` to ensure your solution is correct.
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_list-of-goods/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://saperzp.github.io/react_list-of-goods-hooks/) and add it to the PR description.
```

Comment:
`Replace `<your_account>` with your Github username in the [DEMO LINK](https://saperzp.github.io/react_list-of-goods/) and add it to the PR description.`